functions/scheduled.js

```diff
@@ -0,0 +1,64 @@
+const admin = require("firebase-admin");
+
+const functions = require("firebase-functions");
+const { HttpsError } = require("firebase-functions/lib/providers/https");
+
+const { chunk, sampleSize } = require("lodash");
+
+const DEFAULT_RECENT_USERS_IN_VENUE_CHUNK_SIZE = 6;
```

Comment:
Part of `settings.ts`?

Reply:
I can't move it because it's inside its own `functions` project